Monika After Story/game/script-story-events.rpy

```diff
@@ -1471,6 +1471,86 @@ label mas_steam_install_detected:
     m 5esu "I'd really appreciate it if you would do that for me."
     return
 
+default persistent._mas_pm_has_rpy = None
+init 1 python:
```

Comment:
Can you move this to the `init 970` block in `script-ch30`, and instead just make it just set a flag to True if the conditions match?

Also make sure that `rpyCheckStation` is removed from the global symbol table (or do this in a way that doesn't require storing a temporary variable) so later code wont access this docking station by accident. `postbday_ev` in the `970` init block gets deleted there so you can use that for reference.